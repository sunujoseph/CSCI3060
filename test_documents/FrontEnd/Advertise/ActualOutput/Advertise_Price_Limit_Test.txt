Enter Username: Enter Item Name: Enter Minimum Bid: Error: Maximum Price For Item Is 999.99
Enter Minimum Bid: Error: Invalid Input
Error: Maximum Price For Item Is 999.99
Enter Minimum Bid: 