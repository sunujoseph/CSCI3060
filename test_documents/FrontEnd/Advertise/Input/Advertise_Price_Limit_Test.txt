login
userSS
advertise
New Item
20000000
