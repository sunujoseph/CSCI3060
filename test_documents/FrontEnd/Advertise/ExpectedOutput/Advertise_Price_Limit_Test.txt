Enter Username:
Enter Item Name: 
Enter Minimum Bid: 
Error: Maximum Price For Item Is 999.99
Enter Minimum Bid: 